interface-definitions/interfaces_wireless.xml.in

```diff
@@ -443,10 +558,14 @@
               </valueHelp>
               <valueHelp>
                 <format>u32:34-173</format>
-                <description>5Ghz (802.11 a/h/j/n/ac) Channel</description>
+                <description>5Ghz   (802.11 a/h/j/n/ac) Channel</description>
```

Comment:
extra whitespace

Reply:
This is resolved in commit [e6b896d](https://github.com/vyos/vyos-1x/pull/3524/commits/e6b896d4987c4ce474970fc4703216405d031e34)